src/server/rpc/procedures/google-maps/google-maps.js

```diff
@@ -206,9 +206,39 @@ StaticMap.prototype.getLatitude = function(y){
     });
 };
 
+/**
+ * converts image coordinates to earth coordinates.
+ * @param {Number} x x position of the point on the stage
+ * @param {Number} y y position of the point on the stage
+ * @returns {Array} latitude longitude of the point as a list.
```

Comment:
I would rephrase this to say "A list containing the latitude and longitude of the given point"